test_rclcpp/test/node_check_names.cpp

```diff
@@ -12,7 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <algorithm>
 #include <string>
+#include <utility>
+#include <vector>
```

Comment:
Because `auto` is used and `vector` isn't I think this include could be omitted.

Reply:
I thought you still had to include any types that `auto` would need? Still, `vector` isn't being used anyways so out it goes.